umad-06-payreq-response.md

```diff
@@ -17,7 +17,12 @@ The full structure of the LNURLP response is:
     "utxos": string[],
     // A url which the sending VASP should call on transaction completion to notify the receiving VASP of
     // the utxos used to complete the transaction. See [UMAD-07](/umad-07-post-tx-hooks.md).
-    "utxoCallback": string
+    "utxoCallback": string,
+    // The receiving VASP's signature over sha256_hash(<sender UMA> (eg. $alice@vasp1.com) + <receiver UMA> (eg. $bob@vasp2.com) +
```

Comment:
Do we easily have access to both the sending and receiving UMA on the receiving VASP at this point? I think we should, but it might be good to check out one of the demo VASPs just to make sure this will be straightforward to do in practice without needing to store a bunch of stuff.

Reply:
We have easy access to the sending UMA and the receiving VASP should(??) know the receiver's UMA, but maybe we should include the sender UMA only for simplicity?

Reply:
hmmm now that I think about it, I think VASPS will need to have the receiver uma anyway at this point one way or another for their own internal ledgering. Sorry for the back-and-forth on this.

Reply:
actually let's remove it. jk, lgtm.